integration-tests/bootstrap.user.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
```

Comment:
One thing we have found useful in shell scripts is to add this line:
`set -uxe` 
at line 2 which will do some handy things described [here](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html)